src/viur/core/skeleton.py

```diff
@@ -408,7 +408,16 @@ def setBoneValue(cls, skelValues: t.Any, boneName: str, value: t.Any,
         bone = getattr(skelValues, boneName, None)
         if not isinstance(bone, BaseBone):
             raise ValueError(f"{boneName} is no valid bone on this skeleton ({skelValues})")
-        skelValues[boneName]  # FIXME, ensure this bone is unserialized first
+        if value is None:
+            if append:
+                raise ValueError("Can not append None")
+            if language:
+                skelValues[boneName][language] = None
+            else:
+                skelValues[boneName] = None
```

Comment:
Shouldn't this better be a list on multiple bones?
```suggestion
                skelValues[boneName][language] = [] if bone.multiple else None
            else:
                skelValues[boneName] = [] if bone.multiple else None
```